File: 1925-count-square-sum-triples/1925-count-square-sum-triples.cpp
class Solution {
public:
    int countTriples(int n) {
        int res = 0;
        for(int i = 1; i < n; i++)
        {
            for(int j = i + 1; j < n; j++)
            {
                double c = sqrt(pow(i, 2) + pow(j, 2));
                if(c == ceil(c) && c <= n)
                {
                    res += 2;
                }
            }
        }
        return res;
    }
};

File: 1925-count-square-sum-triples/1925-count-square-sum-triples.py
class Solution:
    def countTriples(self, n: int) -> int:
        res = 0
        for i in range(1, n):
            for j in range(i + 1, n):
                c = sqrt(i**2 + j**2)
                if c == ceil(c) and c <= n:
                    res += 2
        return res